lib/common/options_display.c

```diff
@@ -130,8 +133,12 @@ option_list_default(pcmk__output_t *out, va_list args)
     const uint32_t filter = va_arg(args, uint32_t);
     const pcmk__cluster_option_t *option_list =
         va_arg(args, pcmk__cluster_option_t *);
+    const bool all = (bool) va_arg(args, int);
 
-    bool old_fancy = false;
+    const bool old_fancy = false;
```

Comment:
const snuck in this commit and got removed in the next

Reply:
Whoops, thanks